### ISSUE TITLE
Make sure JusticeHub complies with Section 508 of the Rehabilitation Act of 1973

### ISSUE THREAD

**Issue by christyleos** [TechForJustice/JusticeHub#19]
Section 508 was enacted to eliminate barriers to information technology for people with disabilities by requiring federal government agencies to provide comparable information and data to individuals with or without disabilities. The site should be reviewed periodically to ensure that it meets and exceeds the requirements of Section 508.

**Accessibility Testing Protocols**
When testing and repairing individual web pages the JusticeHub team will use tools, such as the W3C HTML Validator, WAVE by WebAIM and the Color Contrast Analyzer, to ensure the best experience possible for all visitors.

The primary recommended validation tool is WAVE by WebAIM at http://wave.webaim.org/. It reports accessibility violations by annotating a copy of the page that was evaluated and at the same time, providing recommendations on how to repair them. Rather than providing a complex technical report, WAVE shows the original web page with embedded icons and indicators that reveal the accessibility information within the page.
